### PR TITLE
tukan/cicada - unify scrollbar colors to theme

### PR DESCRIPTION
**Pull request by opnsenseuser**
firefox/chrome/safari

note: some css properties are not or not yet supported in firefox

### CODE REVIEW

File: misc/theme-cicada/Makefile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		1.12
+PLUGIN_VERSION=		1.12_2
```
Comment (fabianfrz) [inline misc/theme-cicada/Makefile]: to do this, use PLUGIN_REVISION

Comment (fichtner) [reply misc/theme-cicada/Makefile]: better clear it, this looks more like a feature anyway (PLUGIN_VERSION=1.13 and delete PLUGIN_REVISION)

File: misc/theme-tukan/Makefile [hunk 2]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-tukan
-PLUGIN_VERSION=		1.11
+PLUGIN_VERSION=		1.11_1
```
Comment (fichtner) [inline misc/theme-tukan/Makefile]: same here :)

Comment (fabianfrz) [inline misc/theme-tukan/Makefile]:
```suggestion
PLUGIN_VERSION=		1.11
PLUGIN_REVISION=		1
```

File: misc/theme-cicada/Makefile (hunk 1 above)
Comment (fabianfrz) [reply misc/theme-cicada/Makefile]:
```suggestion
PLUGIN_VERSION=		1.13
```

also possible directly as a delta :)

File: misc/theme-tukan/Makefile (hunk 2 above)
Comment (fabianfrz) [reply misc/theme-tukan/Makefile]:
```suggestion
PLUGIN_VERSION=		1.12
```

**Comment by opnsenseuser** [comment]
I do not care in which version I should change it.
It would only be important to me to which you agree now! :-)
Should I upload it again?
best regards

**Comment by fichtner** [comment]
You can accept Fabian's suggestions and it'll be fixed automatically.

**Comment by fabianfrz** [comment]
@opnsenseuser just click on the green button then commit and the change will be committed automatically. You don't have to do anything locally. @fichtner wants the minor changes (1.12 and 1.13)

File: misc/theme-cicada/Makefile [hunk 3]
```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		theme-cicada
-PLUGIN_VERSION=		1.12
+PLUGIN_VERSION=		1.13
 PLUGIN_REVISION=	1
```
Comment (fichtner) [inline misc/theme-cicada/Makefile]:
```suggestion
```

Comment (fichtner) [reply misc/theme-cicada/Makefile]: still need to remove revision and then it can be merged 👍

**Comment by fichtner** [comment]
merged, thanks!

**Comment by opnsenseuser** [comment]
👍

**Pull request merged**